libs/sdk-ui-loaders/src/dashboard/useDashboardLoader.ts

```diff
@@ -29,6 +29,25 @@ const InitialStatus: DashboardLoadStatus = {
     status: "loading",
 };
 
+const getDashboardConfig = (
+    config: DashboardConfig = {},
+    allowInProgressFeatures: IDashboardLoadOptions["allowInProgressFeatures"],
+): DashboardConfig => {
+    if (allowInProgressFeatures === "alwaysAllow" || allowInProgressFeatures === "staticOnly") {
+        return {
+            ...config,
+            allowInProgressFeatures: true, // will be turned off in case of external plugins later during load
+        };
+    }
+    if (allowInProgressFeatures === "alwaysPrevent") {
+        return {
+            ...config,
+            allowInProgressFeatures: false,
+        };
+    }
+    return config;
```

Comment:
Should this be an error? I mean the allowInProgressFeatures should be always defined at this point, right?

Reply:
fixed